.github/workflows/file-draf-issues.yml

```
@@ -0,0 +1,28 @@
+on:
+  workflow_dispatch:
+
+name: Submit draft issue reports from Strudy analysis
+jobs:
+  update:
+    runs-on: ubuntu-latest
+    steps:
+    - name: Setup node.js
+      uses: actions/setup-node@v2
+      with:
+        node-version: 14
```

Long term support version of Node is currently 16, any reason to stick to 14 for this new job?

```suggestion
        node-version: 16
```